src/main/java/seedu/address/logic/commands/AddExamCommand.java

```diff
@@ -0,0 +1,73 @@
+package seedu.address.logic.commands;
```

Comment:
I think put the addxxxcommands into commands.add package. Else the command folder looks overloaded